Resources/public/css/sb-admin-2.css

```diff
@@ -0,0 +1,354 @@
+/*!
```

Comment:
Those assets are not handle by the PHP World :)
I just "symlinked" in web, no compilation and no rewriting, and i do no have a node stack installed.